src/gui/editors/PianoRoll.cpp

```diff
@@ -130,7 +130,7 @@ QPixmap* PianoRoll::s_toolKnife = nullptr;
 
 SimpleTextFloat * PianoRoll::s_textFloat = nullptr;
 
-static std::array<QString, 12> s_noteStrings {"C", "C# / D\u266D", "D", "D# / E\u266D", "E", "F", "F# / G\u266D", "G", "G# / A\u266D", "A", "A# / B\u266D", "B"};
+static std::array<QString, 12> s_noteStrings {"C", "C\u266F / D\u266D", "D", "D\u266F / E\u266D", "E", "F", "F\u266F / G\u266D", "G", "G\u266F / A\u266D", "A", "A\u266F / B\u266D", "B"};
```

Comment:
could be static const

Reply:
I agree, but it was already non-const, so I don't think it needs to be changed in this PR. My concern would be more with the 186-character line - it could do with being broken in a few places.

Reply:
If we do want a style fixup:
```suggestion
static std::array<QString, 12> s_noteStrings {
	"C", "C\u266F / D\u266D", "D", "D\u266F / E\u266D", "E", "F", "F\u266F / G\u266D", "G", "G\u266F / A\u266D", "A", "A\u266F / B\u266D", "B"
};
```